start-finalSetupWorld

```diff
@@ -25,7 +25,12 @@ if [[ "$WORLD" ]] && ( isTrue "${FORCE_WORLD_COPY}" || [ ! -d "$worldDest" ] );
     mkdir -p /tmp/world-data
     (cd /tmp/world-data && unzip -o -q "$zipSrc")
 
-    baseDirs=$(find /tmp/world-data -name "level.dat" -exec dirname "{}" \;)
+    if [ "$TYPE" = "SPIGOT" ]; then
+      baseDirs=$(find /tmp/world-data -name "level.dat" -exec dirname "{}" \;|grep -v '_nether\|_the_end')
```

Comment:
This code seems totally fine, but am wondering if it would work and streamline it do filter within `find` using the following. While looking that up I noticed the `-printf` action might be handy. So together could be:

```suggestion
      baseDirs=$(find /tmp/world-data -name "level.dat" -not -path "*_nether*" -not -path "*_the_end*" -printf "%h")
```